C/CImGuiPack/build_tarballs.jl

```diff
@@ -80,4 +80,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"8")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.9", preferred_gcc_version=v"8")
```

Comment:
This doesn't do anything, the problem is `julia_versions`

Reply:
Mmmm I see.

Reply:
Should be filtered out in 6ab08098.

Reply:
To be clear, I don't mind _too_ much about this compat, what I meant is that it doesn't affect the "platforms" for which you actually build, but perhaps now that you're building only for julia v1.9+ keeping the compat to v1.9 would be more consistent.

Reply:
Ok, added it back in b1b9f3a0.